### PR TITLE
Add service account detail page & card width update

### PR DESCRIPTION
**Pull request by madhushreeray30**
Screenshot 2023-02-03 at 2 25 50 PM

1845FCE4-3346-430B-AD8B-7CB99E80AB11_1_105_c

### CODE REVIEW

File: dashboards/compute/compute_edges.sp [hunk 1]
```diff
@@ -150,6 +150,25 @@ edge "compute_disk_to_kms_key_version" {
...
+
+  sql = <<-EOQ
+    select
+      id::text as from_id,
+      t as to_id
+    from
+      gcp_compute_firewall,
+      jsonb_array_elements_text(target_service_accounts) as t
+    where
+      id::text = any($1)
```
Comment (misraved) [inline dashboards/compute/compute_edges.sp:166]:
```suggestion
      id::text = any($1);
```

**Review by misraved — CHANGES_REQUESTED** [review]
@madhushreeray30 please take a look at the review comments. Thanks!!

**Review by rajlearner17 — CHANGES_REQUESTED** [review]
We need to add the Service Account node in related resources

**Review by rajlearner17 — APPROVED** [review]
Looks good

**Review by misraved — APPROVED** [review]

**Pull request merged**